src/plugins/intel_cpu/src/nodes/kernels/x64/brgemm_kernel.cpp

```diff
@@ -158,6 +169,19 @@ void BrgemmKernel::init_brgemm(brgemmCtx& ctx,
         THROW_ERROR("cannot be executed due to invalid brgconv params");
     }
 
+    if (use_amx && b_accumulate) {
+        brgemm_attr_t brgattr;
+        brgattr.max_bs = 1;
```

Comment:
Could you add comments why we need this configuration ?

Reply:
Comments added.

Reply:
Does this has side effect when the machine has no AMX support ?

Reply:
No, non-amx will use `vaddps` to complete accumulation.